packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -1774,21 +1785,21 @@ describe("BnsConnection", () => {
     // directly. So let's just check the last information per ID.
     const latestEventPerId = new Map<string, AtomicSwap>();
     for (const event of liveView.value()) {
-      latestEventPerId.set(toHex(event.data.id), event);
+      latestEventPerId.set(toHex(event.data.id.data), event);
```

Comment:
Arg, here we need swap ID serialization again. Could we at least make this explicit in this test file (which is BNS only):

```ts
function serializeBnsSwapId(id: SwapId): string {
  return toHex(id.data);
}
```